examples/apps/presence-tracker/package.json

```diff
@@ -45,9 +45,15 @@
 		"@fluidframework/container-definitions": "workspace:~",
 		"@fluidframework/container-runtime-definitions": "workspace:~",
 		"@fluidframework/core-interfaces": "workspace:~",
+		"@fluidframework/devtools": "workspace:~",
 		"@fluidframework/driver-definitions": "workspace:~",
 		"@fluidframework/fluid-static": "workspace:~",
+		"@fluidframework/presence": "workspace:~",
+		"@fluidframework/routerlicious-driver": "workspace:~",
 		"@fluidframework/runtime-utils": "workspace:~",
+		"@fluidframework/telemetry-utils": "workspace:~",
+		"@fluidframework/test-runtime-utils": "workspace:~",
+		"axios": "^1.7.7",
```

Comment:
It's used in the azure function token provider, which I had removed. I think that token provider
is needed in order to target FRS. I'm using the external-controller example as a guide - let me know if you
think there's a better pattern to follow. I also filed a task to investigate why we have/need multiple copies of that
token provider.

Reply:
Hmmm, maybe we can use InsecureTokenProvider since this is an unpublished example?

Reply:
Refreshing my memory on this stuff... I think if we want to support local or _remote_ AFR instances, then we'll need the AF
token provider. I think that token provider could use fetch instead of axios pretty easily, but not planning to tackle that.